test/quantization/pt2e/test_quantize_pt2e_qat.py

```diff
@@ -608,6 +615,79 @@ def get_source_fn(node: torch.fx.Node):
         self.assertTrue("backbone" in get_source_fn(second_conv))
         self.assertTrue("backbone" in get_source_fn(second_relu))
 
+    def test_qat_conv_bn_bias_derived_qspec(self):
+        m = TestHelperModules.ConvWithBNRelu(relu=False)
+        example_inputs = (torch.randn(1, 3, 5, 5),)
+        m = capture_pre_autograd_graph(m, example_inputs)
+        quantizer = ConvBnDerivedBiasQuantizer()
+        m = prepare_qat_pt2e(m, quantizer)
+        m(*example_inputs)
+        m = convert_pt2e(m)
+        m(*example_inputs)
+
+
+class ConvBnDerivedBiasQuantizer(Quantizer):
```

Comment:
should we add a test for referring to bias node as well? e.g. qspec for weight being shared with bias